src/roadstop.cpp

```diff
@@ -369,6 +381,7 @@ void RoadStop::Entry::Rebuild(const RoadStop *rs, int side)
 	this->occupied = 0;
 	for (const auto &it : rserh.vehicles) {
 		this->occupied += it->gcache.cached_total_length;
+		this->vehicles += 1;
```

Comment:
Shouldn't we set `this->vehicles = 0;` before the loop, to match `this->occupied = 0;`?